covid_graphs/covid_graphs/country_graph.py

```diff
@@ -59,19 +65,34 @@ def adjust_xlabel(date: datetime.date):
 
         colors = ["SteelBlue", "Purple", "Green", "Orange", "Gray"][: len(self.curves)]
 
-        # Traces
-        traces = []
+        traces, shapes = [], []
```

Comment:
I was thinking we should maybe make `_get_traces()` and `_get_shapes()` functions. Of yes, we can do another PR.